apps/website/src/pages/show-and-tell/index.tsx

```diff
@@ -391,6 +401,9 @@ const ShowAndTellIndexPage: NextPage<ShowAndTellPageProps> = ({
           onKeyDown={handleArrowKeys}
           tabIndex={-1}
         >
+          <h1 className="text-center text-3xl italic">
+            {totalPostsCount} posts by {usersCount} users
+          </h1>
```

Comment:
```suggestion
          <p className="text-center text-3xl italic">
            {totalPostsCount} posts by {usersCount} users
          </p>
```
Semantically, I don't think this should be an h1, it is not the primary heading for the entire page. I don't think it should be a semantic heading at all.